### PR TITLE
Add enpt

### PR DESCRIPTION
**Pull request by danschef**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml"
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged)
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details)
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

**Comment by danschef** [comment]
The checks currently fail because some dependencies are not yet on conda-forge. Do I have to deal with this or is this automatically handles somehow?

**Comment by danschef** [comment]
@conda-forge-admin, please ping team

**Comment by synapticarbors** [comment]
The missing packages will have to be built on conda-forge. Have a look at https://github.com/marcelotrevisani/grayskull, which can make this easier.

**Comment by danschef** [comment]
Ok, so I have to add the recipes for the missing packages to the current PR?

What about the mvgavg dependency? This is a third-party dependency (not developed by me) which could not be resolved during the build check (see [here](https://dev.azure.com/conda-forge/feedstock-builds/_build/results?buildId=216934&view=logs&jobId=10bda42a-d420-5c28-a43f-bc4e66405e5c&j=10bda42a-d420-5c28-a43f-bc4e66405e5c&t=bef775d0-a1a2-55a3-9721-17147029baf0)). This package is not yet on conda-forge but installs easily from pip.

**Comment by synapticarbors** [comment]
To give you a hand, I've added a PR for `mvavg` here: https://github.com/conda-forge/staged-recipes/pull/12778

**Comment by danschef** [comment]
@synapticarbors Could you give me some advice whats wrong with line 25 in meta.yaml of sensormapgeo?

Here is it:
    - _openmp_mutex =* =*llvm* [Unix]

With this line I want to make sure that the llvm build of _openmp_mutex is used for Linux.

File: recipes/sensormapgeo/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+
+requirements:
+  host:
+    - pip
+    - python
+    - setuptools
+  run:
+    - gdal
+    - numpy
+    - _openmp_mutex =* =*llvm* [Unix]
```
Comment (chrisburr) [inline recipes/sensormapgeo/meta.yaml:26]:
```suggestion
    - _openmp_mutex =* =*llvm*  # [unix]
```

File: recipes/sensormapgeo/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+
+requirements:
+  host:
+    - pip
+    - python
+    - setuptools
+  run:
+    - gdal
+    - numpy
+    - _openmp_mutex =* =*llvm*  # [unix]
```
Comment (isuruf) [inline recipes/sensormapgeo/meta.yaml:26]: See https://conda-forge.org/docs/maintainer/knowledge_base.html#openmp

Comment (danschef) [reply recipes/sensormapgeo/meta.yaml:26]: Thanks for the hint. So is fine to use the `_openmp_mutex=*=*_llvm` line in the run requirements sections or should I better add `- libgomp      # [linux]` to the build section?

Comment (isuruf) [reply recipes/sensormapgeo/meta.yaml]: Just
```
# meta.yaml
requirements:
  build:
    - llvm-openmp  # [osx]
    - libgomp      # [linux]
```

File: recipes/sensormapgeo/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+  sha256: 4f435416cb98602882e0a122a5499b8ffb027a36b6eae5529e493ae235a5cfcd
+
+build:
+  number: 0
+  skip: true  # [py<36]
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  build:
+    # force the use of llvm to fix https://gitext.gfz-potsdam.de/EnMAP/sensormapgeo/-/issues/6
```
Comment (isuruf) [inline recipes/sensormapgeo/meta.yaml:20]: Ah, I didn't realize that you actually wanted LLVM openmp. Then what you had was correct.

**Comment by danschef** [comment]
The checks are currently failing but I can't really figure out what causes the problem. The logs are full of package incompatibilities which can't be true as we have all the packages running here.

Any idea how to solve that?

**Comment by danschef** [comment]
@conda-forge-admin, please ping team

I think I need some assistance here. The sicor package does not build and I cannot figure out the reason for that. The logs contain a lot of package conflicts but without pointing to a specific library. Could that be a problem with conda-build?

**Comment by scopatz** [comment]
Seems as though there are conflcits now

**Comment by danschef** [comment]
Could sicor be declared as a noarch package? Maybe that would help. sensormapgeo is already merged in the meantime. I will remove the recipe. pyrsr should also be merged soon.

**Comment by danschef** [comment]
@conda-forge-admin, please restart ci

**Comment by danschef** [comment]
@conda-forge-admin, please rerender

**Comment by danschef** [comment]
@conda-forge/staged-recipes I removed the recipes fpr pyrsr and sensormapgeo as they have already been merged in separate PRs. I declared sicor and enpt as noarch packages but the CI still runs on Linux, OSX and Windows. It still fails due to package conflicts without pointing to a specific library.

Can anybody help?

**Comment by dopplershift** [comment]
So noarch won't affect what CIs run here--we can just ignore the results on mac/windows if they're not supposed to succeed.

I'm not sure what's up with the conflict. I can seem to get the run-list of packages from sicor to install locally on my mac.

**Comment by danschef** [comment]
I think I could narrow it down to basemap which is a dependency of arosics. I will fix this in the next version of arosics and then hopefully the build runs through.

**Comment by danschef** [comment]
@conda-forge-admin, please restart ci

**Comment by danschef** [comment]
@conda-forge/staged-recipes

Seems like I finally managed to get the build checks running properly. So this ready for the final review now.

**Comment by danschef** [comment]
@conda-forge-admin, please ping team

File: recipes/enpt/meta.yaml [hunk 4]
```diff
@@ -0,0 +1,62 @@
...
+    - utm
+
+test:
+  imports:
+    - enpt
+    - enpt.execution
+  # commands:
+    # - pip check  # disabled as sicor requirements contain test and docs requirements that are not installed with the conda-forge package
+  # requires:
+    # - pip
```
Comment (scopatz) [inline recipes/enpt/meta.yaml:52]: Please remove comments

Comment (danschef) [reply recipes/enpt/meta.yaml]: Done.

**Comment by danschef** [comment]
Thanks a lot @scopatz!

**Pull request merged**